spec/requests/file_attachments_spec.rb

```diff
@@ -279,22 +279,38 @@
   end
 
   describe "PATCH /documents/:document/file-attachments/:file_attachment_id/edit" do
+    let(:document_type) { build(:document_type, attachments: "featured") }
+    let(:file_attachment_revision) { create(:file_attachment_revision) }
+    let(:edition) do
+      create(:edition,
+             file_attachment_revisions: [file_attachment_revision],
+             document_type: document_type)
+    end
+
     it "redirects to the featured attachments index page" do
       stub_any_publishing_api_put_content
       stub_asset_manager_receives_an_asset
 
-      file_attachment_revision = create(:file_attachment_revision)
-      document_type = build(:document_type, attachments: "featured")
-      edition = create(:edition,
-                       file_attachment_revisions: [file_attachment_revision],
-                       document_type: document_type)
-
       patch edit_file_attachment_path(edition.document,
                                       file_attachment_revision.file_attachment_id),
             params: { file_attachment: { unique_reference: "Uniq Ref" } }
 
       expect(response).to redirect_to(featured_attachments_path(edition.document))
     end
+
+    it "returns issues and an unprocessable response when there are requirement issues" do
+      max_length = Requirements::FileAttachmentMetadataChecker::UNIQUE_REF_MAX_LENGTH
+      too_long_unique_reference = "a" * (max_length + 1)
```

Comment:
Minor, but in some tests we're using `z` and in others we're using `a`. Would be better to stick to one!

Reply:
Hmm...this didn't cause any pain for me, really, so I'm going to leave as-is. We can always come back to it if it becomes a source of confusion.